src/seedu/addressbook/commands/HappyCNY.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+/**
+ * Clears the address book.
+ */
+public class HappyCNY extends Command {
+
+    public static final String COMMAND_WORD = "HappyCNY";
+    public static final String MESSAGE_USAGE = "It is not CNY.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book is greeting you!";
+
+    public HappyCNY() {
```

Comment:
If the constructor does not perform any meaningful functionality, you can remove it.